docs/maps/vector-layer.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[vector-layer]]
+== Vector layer
+
+Display points, lines, and polygons.
+
+[role="screenshot"]
+image::maps/images/vector_layer.png[]
+
+Vector layers can be created from the following sources
+
+*Custom vector shapes*:: Vector shapes from static files configured in kibana.yml.
```

Comment:
possible to add link to `docs/setup/settings.asciidoc` page (?) and mention `map.regionmap.*` config

Reply:
<img width="783" alt="screen shot 2019-02-28 at 3 14 04 pm" src="https://user-images.githubusercontent.com/373691/53602286-89be4b80-3b6b-11e9-8b50-bfc77de6a0d3.png">